src/App.css

```diff
@@ -27,8 +27,22 @@
 .addColorInputBtnContainer {
   display: flex;
   margin: 0.5rem 0 2rem;
+  column-gap: 0.5rem;
```

Comment:
These changes are from the previous PR, I think you need to fetch upstream and merge it into your branch.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix